README.md

```diff
@@ -12,20 +12,31 @@ Requirements
 * Amazon IAM Access Key ID/Secret Access Key
   * which must have the access lights of the target Amazon S3 bucket.
 
+OR
+
+* Google Cloud Interoperable storage access keys (see https://cloud.google.com/storage/docs/migrating#keys)
+  * GCP_ACCESS_KEY_ID, GCP_SECRET_ACCESS_KEY, and GCP_PROJECT_ID are only required if using HMAC authentication.
+  * When using oauth authentication, a docker mount ` -v ~:/mab` can be added to save auth0 credentials to your home directory after mongodb-awesome-backup is run.  On subsequent runs, the same `~/.bota` file will be used for authentication.
```

Comment:
Perhaps, is `mab` the abbreviation of "Mongodb Awesome Backup"?

And is `~/.bota` mistake of `~/.boto`??

Reply:
Ha, ha!  Yes, you are right... bota should be boto!  Fixed it.  I did that more than once during the execution of this PR.  As you're 2 for 2, as mab is the abbreviation of "Mongodb Awesome Backup".  Cheers!

Reply:
Hmm...I think users cannot read the meaning of "mab" from this document like me...

Could you add an explanation of "mab"?
This is the final change request.

Reply:
Done